libraries/AP_Filesystem/AP_Filesystem_Mission.cpp

```diff
@@ -412,6 +412,9 @@ bool AP_Filesystem_Mission::finish_upload(const rfile &r)
             }
         }
     }
+    if (!mission->check_do_jump_tags()) {
```

Comment:
Isn't this too-late?!

Reply:
No. The problem is a jump tag can be after a do jump tag so there's no way to correctly verify it until you have the whole mission